### PR TITLE
[Feature]: json_schema in response support for Anthropic 

### LINKED ISSUE #6741: [Feature]: json_schema support for Anthropic
### The Feature

Currently, you only support a small amount of `json` format models :
https://docs.litellm.ai/docs/completion/json_mode

### Motivation, pitch

I would need to be able to do the same with Anthropic models, without having to specify to the model how to output in json

### Twitter / LinkedIn details

_No response_

### PR DESCRIPTION
**Pull request by ishaan-jaff**
## Type

🆕 New Feature
🐛 Bug Fix
🧹 Refactoring
📖 Documentation
🚄 Infrastructure
✅ Test

## [REQUIRED] Testing - Attach a screenshot of any new tests passing locall
If UI changes, send a screenshot/GIF of working UI fixes

### CODE REVIEW

File: litellm/llms/anthropic/chat/handler.py [hunk 1]
```diff
@@ -318,6 +325,37 @@ def _process_response(
         model_response._hidden_params = _hidden_params
         return model_response
 
+    @staticmethod
+    def _convert_tool_response_to_message(
```
Comment (ghost) [inline litellm/llms/anthropic/chat/handler.py:329]: @ishaan-jaff this code should **not** be in the handler function as it's about llm translation.

it should be in the transformation.py or prompt_factory file 

---
acknowledging prior code was in handler - it seems like that was missed during a refactor.

Comment (ishaan-jaff) [reply litellm/llms/anthropic/chat/handler.py:329]: hey I think a lot of the previous code for this was written by you @krrishdholakia

Comment (ishaan-jaff) [reply litellm/llms/anthropic/chat/handler.py:329]: This is the PR I looked at - had no idea why this logic was in the handler to begin with but assumed there must have been a reason you had https://github.com/BerriAI/litellm/commit/f2401d6d5eadef39036feacb034dba6f5e102a74#diff-4b817e67059fdef6b8fabcd1e356d6597ffdde6ff251a1233a01d41b8805207b

Comment (ishaan-jaff) [reply litellm/llms/anthropic/chat/handler.py:329]: fixed here @krrishdholakia https://github.com/BerriAI/litellm/pull/6834

**Pull request merged**